docs/helm-chart/production-guide.rst

```diff
@@ -52,6 +52,9 @@ configure the secret name:
 
 .. _production-guide:pgbouncer:
 
+.. warning::
+  If you use `CeleryExecutor`, keep in mind that `resultBackendSecretName` expects a url that starts with `db+postgresql://`, while `metadataSecretName` expects `postgresql://` and wont work with `db+postgresql://`. You'll need to create separate secrets with the correct schema.
```

Comment:
```suggestion
  If you use `CeleryExecutor`, keep in mind that `resultBackendSecretName` expects a url that starts with `db+postgresql://`, while `metadataSecretName` expects `postgresql://` and wont work with `db+postgresql://`. You'll need to create separate secrets with the correct scheme.
```

Reply:
Should `CeleryKubernetesExecutor` be added as well ?
I notice the docs usually mention only one of them.

Reply:
We normally just reference CeleryExecutor explicitly, as CeleryKubernetesExecutor actually runs a CeleryExecutor under the hood. Said another way, it's inferred (at least to me).